### PR TITLE
ZENKO-3111 setting up release stage for s3utils

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

File: docs/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+docker pull registry.scality.com/s3utils/s3utils:<tag>
+```
+
+## Release Process
+
+To release a production image:
+
+* Name the tag for the repository and Docker image.
+
+* Use the `yarn version` command with the same tag to update `package.json`.
```
Comment (miniscruff) [inline docs/RELEASE.md:31]: ooh I didn't know about yarn/npm version... _keeps notes_

Comment (tcarmet) [reply docs/RELEASE.md:31]: me neither! I took this from @anurag4DSB's note on Vault.

**Review by miniscruff — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**